plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt_test.go

```diff
@@ -96,6 +100,176 @@ func TestHandleTelemetryTwoSimple(t *testing.T) {
+											ValueByType: &telemetryBis.TelemetryField_StringValue{StringValue: status.stringValue},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		expected: []telegraf.Metric{
+			metric.New(
+				"deleted",
+				map[string]string{
+					encodingPath.name: encodingPath.stringValue,
+					name.name:         name.stringValue,
+					index.name:        index.stringValue,
+					ip.name:           ip.stringValue,
+					source.name:       source.stringValue,
+					subscription.name: subscription.stringValue,
+				},
+				map[string]interface{}{
+					deleteKey:              false,
+					ip.fieldName:           ip.stringValue,
+					prefixLength.fieldName: prefixLength.uint64Value,
+					origin.fieldName:       origin.stringValue,
+					status.fieldName:       status.stringValue,
+				},
+				time.Now(),
+			)},
+	},
+		{
+			telemetry: &telemetryBis.Telemetry{
+				MsgTimestamp: 1543236572000,
+				EncodingPath: encodingPath.stringValue,
+				NodeId:       &telemetryBis.Telemetry_NodeIdStr{NodeIdStr: source.stringValue},
+				Subscription: &telemetryBis.Telemetry_SubscriptionIdStr{SubscriptionIdStr: subscription.stringValue},
+				DataGpbkv: []*telemetryBis.TelemetryField{
+					{
+						Delete: true,
+						Fields: []*telemetryBis.TelemetryField{
+							{
+								Name: "keys",
+								Fields: []*telemetryBis.TelemetryField{
+									{
+										Name:        name.name,
+										ValueByType: &telemetryBis.TelemetryField_StringValue{StringValue: name.stringValue},
+									},
+									{
+										Name:        index.name,
+										ValueByType: &telemetryBis.TelemetryField_Uint32Value{Uint32Value: index.uint32Value},
+									},
+									{
+										Name:        ip.name,
+										ValueByType: &telemetryBis.TelemetryField_StringValue{StringValue: ip.stringValue},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			expected: []telegraf.Metric{
+				metric.New(
+					"deleted",
+					map[string]string{
+						encodingPath.name: encodingPath.stringValue,
+						name.name:         name.stringValue,
+						index.name:        index.stringValue,
+						ip.name:           ip.stringValue,
+						source.name:       source.stringValue,
+						subscription.name: subscription.stringValue,
+					},
+					map[string]interface{}{deleteKey: true},
+					time.Now(),
+				)},
+		},
+	}
+	for _, test := range testCases {
+		c := &CiscoTelemetryMDT{
+			Log:                testutil.Logger{},
+			Transport:          "dummy",
+			Aliases:            map[string]string{"deleted": encodingPath.stringValue},
+			IncludeDeleteField: true}
+		acc := &testutil.Accumulator{}
+		// error is expected since we are passing in dummy transport
+		require.ErrorContains(t, c.Start(acc), "dummy")
+		data, err := proto.Marshal(test.telemetry)
+		require.NoError(t, err)
+
+		c.handleTelemetry(data)
+		actual := acc.GetTelegrafMetrics()
+		fmt.Println(test.expected)
+		fmt.Println(actual)
```

Comment:
Can those be removed please
```suggestion
```

Furthermore, please resolve the merge conflict.